OKRs at FlowForge (#559)
flowforge/website: src/handbook/company/achieving-success.md

---

> navTitle: KPI

Author: ```suggestion
navTitle: KPIs and OKRs
```

meta:

>     title: KPI

Author: ```suggestion
    title: KPIs and OKRs
```

## KPI

A KPI, or Key Point Indicator, is used as a heuristic metric for company wide
success. At FlowForge we're engaged in many initiatives to grow our offering and
assess product market fit. At this time there's one KPI to track progress.

### Managed Node-RED instances

[Dashboard](https://datastudio.google.com/reporting/90315614-39bf-4eac-ba3d-45b4c635940e/page/P1V6C)

FlowForge is the key product we're verifying in the market currently. Each
customer electing to use FlowForge to help them in their Node-RED journey is a
signal we're providing value.

An instance is either a remote Node-RED running the [FlowForge Device Agent](https://flowforge.com/docs/user/devices/),
a hosted Node-RED in the FlowForge Cluster, or an otherwise FlowForge licensed 
and supported Node-RED instance.

Telemetry collection is opt-in, and dependant on an internet connection. As such
it's anticipated to under report the true number of FlowForge managed instances.

Target growth for this metric is 10% week over week.

## OKRs

OKRs, Objectives and Key Results, are action oriented goals that have a _measureable_
impact on the company. OKRs are intended to be bold goals, where the bar is set

> high enough that one still takes a leap.

Reviewer: ```suggestion
high enough that one still takes a leap. They are also meant to be a communication tool so that everyone in the company knows what other groups are trying to achieve.
```

Given goals are intended to be bold and aggressive, performance on the OKRs has
no influence on compensation or any other performance indicator of a department
or individual.

### Anatomy of an OKR

Per timeboxed period a couple of OKRs are created. Each of them should describe

> clearly what the objective is, the plan to achieve that goal, and how success is

Reviewer: ```suggestion
clearly what the objective is, the key results that will be used to measure the success.
```

measured. Further, the goal and plan to achieve it should align with the
[values](./values/).

### The process

Before the OKR period, the CEO sets company wide OKRs. Each of the CEO, CTO,
Head of Marketing, Head of UX, and product manager are expected to read the OKRs

> and create 3 OKRs of their own to help to achieve the company wide goal.

Author: ```suggestion
and create 2 OKRs of their own to help to achieve the company wide goal.
```

> and create 2 OKRs of their own to help to achieve the company wide goal.

Reviewer: ```suggestion
and work with their teams to develop at least 2 OKRs their team will implement to help achieve the company goal.
```

> and create 2 OKRs of their own to help to achieve the company wide goal.

Author: ```suggestion
and create 2 OKRs of their own to help to achieve the company wide goal.

A retrospective on the OKRs will be held at the end of the OKR timespan. This will ensure we all
learn about what works and what doesn't for the company.
```

> 

Reviewer: At the start of the OKR period, each team will present their OKRs during a company meeting. Bi-weekly meetings will be held where each team will present a quick update on the status of their OKR and current work being done to achieve the key result.  At the end of the OKR period, a company wide retrospective will he held to share lessons learned from the OKR.

Author: ```suggestion

At the start of the OKR period, each team will present their OKRs during a company meeting. Bi-weekly meetings will be held where each team will present a quick update on the status of their OKR and current work being done to achieve the key result. At the end of the OKR period, a company wide retrospective will he held to share lessons learned from the OKR.

```

### Current OKRs

> [OKRs are tracked here](https://docs.google.com/document/d/12zOCFxot0rlRY-_hNwCmuv7_U1STqjzoaXh8EkIWtZI){rel="nofollow"}.

Reviewer 2: This document isn't accessible

Author: Fixed
